src/clue.js

```diff
@@ -117,10 +117,29 @@ var Patio = {name: 'Patio'}
 
 
 // Characters Collection
-var charactersArray = [drOrchid, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
+var charactersArray = ['mrGreen', 'drOrchid', 'profPlum', 'missScarlet', 'mrsPeacock', 'mrMustard'];
```

Comment:
Here the arrays are being created with the name of the objects as strings. The objects should be added without quotation marks so you have the whole object with the information inside them.